### DISCUSSION TITLE
How to setup API for Airbyte OSS.

### DISCUSSION THREAD

**Discussion by jschneider98** [airbytehq/airbyte#40552] [Platform API Questions]
I can't find any documentation on how to setup the API with OSS. The docs say it's available, but I can only find documentation on setting it up with Enterprise/Cloud. Is there some documentation for OSS API setup that I'm missing?

**Comment by nataliekwong** [comment]
Hey @jschneider98 , we're currently working with our API docs provider, ReadMe, to restore a configuration page that explains how to get started with the Airbyte API. It's currently missing from our documentation on api.airbyte.com. We're sorry about the temporary inconvenience, and I'll update here when it's back!

**Reply by jschneider98** [reply]
Thank you so much!

**Comment by nataliekwong** [comment]
Hey there, the docs around using the Airbyte API can now be found here: https://reference.airbyte.com/reference/getting-started.

Feel free to re-open this if you have questions!

**Reply by jschneider98** [reply]
Perfect. I found the information needed here: [https://reference.airbyte.com/reference/authentication]()

Thanks!
